packages/smoldot-provider/src/SmoldotProvider.ts

```diff
@@ -18,10 +18,13 @@ import { HealthCheckError } from './errors';
 
 const l = logger('smoldot-provider');
 
+// Note types are copied from polkadotjs for consistency - `any` types are used
+// there
+
 interface RpcStateAwaiting {
   callback: ProviderInterfaceCallback;
   method: string;
-  params: any[];
+  params: any[]; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
You should not disable linter. Any should not be used

Reply:
This came from PolkadotJS ... I wanted to keep it the same as the original code intentionally